docs/README.rst

```diff
@@ -130,4 +139,14 @@ Runs all of the stages above in one go: i.e. ``destroy`` + ``converge`` + ``veri
 ``bin/kitchen login``
 ^^^^^^^^^^^^^^^^^^^^^
 
-Gives you SSH access to the instance for manual testing.
+Gives you SSH/RDP access to the instance for manual testing.
+
+Note
+^^^^
+When testing using Vagrant you must set the environment variable ``KITCHEN_LOCAL_YAML`` to ``kitchen.vagrant.yml``.  For example:
+
+.. code-block:: bash
+
+   $ KITCHEN_LOCAL_YAML=kitchen.vagrant.yml bin/kitchen test  # Alternatively,
+   $ export KITCHEN_LOCAL_YAML=kitchen.vagrant.yml
+   $ bin/kitchen test
```

Comment:
The `README` is one file that's not managed by the `ssf-formula`, so propagating changes is not so easy.  Do you think there's a way we could append the information about `kitchen-vagrant` at the end of the `Testing` section?  The one good thing is that this is at the end of each `README`, so it would be a much easier task to automate across these 50+ formulas.  What's your opinion about that?

Reply:
So you mean rather than interspersing some of the changes, grouping them all in a new section at the end, right?

Reply:
If possible, this would ease the burden significantly.  I expect there will be formulas where Vagrant testing won't be applicable (e.g. the `apt-formula`), so keeping things consistent will be easier to manage all round.

Reply:
Thanks for making the changes.